common/tables/index.json-ld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Tables",
+		"fr": "Les Tables"
+	},
+	"description": {
+		"en": "HTML elements used as is.",
+		"fr": "Les éléments HTML utilisé tel quel."
```

Comment:
Need a better descriptive description. 

For the French you can leave it blank and we will provide you its translated version on a subsequent review.

Reply:
Tableaux avec des styles personalisés.